server/@types/dto/index.d.ts

```diff
@@ -77,10 +77,12 @@ declare module 'dto' {
   }
 
   export interface ReviewPlanDto {
+    prisonNumber: string
```

Comment:
I've had to change the ReviewPlanDto slightly to carry the prisonNumber, prisonId, and also make the reviewDate a proper Date field 👍